algorithms/map.go

```diff
@@ -1,5 +1,13 @@
 package algorithms
 
+func MapToSlice[K comparable, V any](m map[K]V) []V {
```

Comment:
I do think this is the same as: https://pkg.go.dev/github.com/samber/lo#Values (we can reimplement though, but it should be named `Values` or `MapValues` as well)